tests/scripts/feature_class_exception.py

```diff
@@ -0,0 +1,24 @@
+# Set logger to debug level
+import logging
+
+from pydantic import BaseModel
+
+from datachain.lib.dc import C, DataChain
+
+logging.basicConfig(level=logging.INFO)
+
+
+class Embedding(BaseModel):
+    value: float
+
+
+ds_name = "feature_class_error"
+ds = (
+    DataChain.from_storage("gs://dvcx-datalakes/dogs-and-cats/")
```

Comment:
can we at least use something simpler - like `from_values` - so that we don't do expensive networking, indexing, etc?